src/app/Components/home/Components/amy-chat/Objects/message.ts

```diff
@@ -1,4 +1,5 @@
 export class Message {
+    isResponse: boolean;
```

Comment:
replace this by `from` so it is more generic

Reply:
yaou say I should rename it to fromBackend or fromAmy? because fromResponse is not what we are checking here, we are checking if the Message is a Response Type Message.